includes/compatibility.php

```diff
@@ -32,7 +31,10 @@ class Compatibility {
 	public static function register_actions() {
 		add_action( 'init', [ __CLASS__, 'init' ] );
 
-		self::polylang_compatibility();
+		self::set_editor_language();
```

Comment:
Please restore the old file structure. In order to make a clear diff.

Reply:
Done